### DISCUSSION TITLE
Crashing while trying to deploy

### DISCUSSION THREAD

**Discussion by tristancamejo** [vercel/vercel#6951] [Help]
I need help on why deploying my next js app keeps crashing.
![ACB2D967-9AC9-4728-B758-66EBE6041DF9](https://user-images.githubusercontent.com/69066026/140511896-d59ae5dc-ea9c-45b2-a25a-8c1a7aed8692.png)
![8DD47A32-8BDF-4834-A540-35CD419D203F](https://user-images.githubusercontent.com/69066026/140511915-1239f46f-1fbb-41bc-bfdf-34a2402a5312.png)

**Comment by leerob** [comment]
Hey! Could you share the error the build sees? Are you seeing the same error locally when running `next build`?

**Reply by tristancamejo** [reply]
Hey, I figured it out running `yarn dev` worked, but running `yarn build` didn't because of a typescript problem. I fixed it now and learnt building isn't the same as testing.
